recipes/kessler/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "kessler" %}
+{% set org = "kesslerlib" %}
```

Comment:
This is not used, right? Also, we try to use jinja variables only when they are extremely necessary, like for the version and name. Otherwise it obscure the recipe and doesn't help much.

```suggestion
```